app/lib/continuousTimeUtils.js

```diff
@@ -28,7 +28,7 @@ function addTimePadding(parsedTimes, isOneSession) {
     if (time.sessions.length < max) {
       // eslint-disable-next-line no-param-reassign
       time.padding = (max - time.sessions.length);
-    } else if ((time.sessions.length === 1) && (isOneSession === true)) {
+    } else if ((time.sessions.length === 1) || (isOneSession === true)) {
```

Comment:
This logic seems to be in place to always force the exceptional opening time to use the single session styling, regardless of how many sessions there are.

This leads to bad formatting on the page, i.e. http://profiles-pr-151.dev.beta.nhschoices.net/gp-surgeries/41823